crud/src/main/java/com/redhat/lightblue/assoc/ep/Assemble.java

```diff
@@ -216,7 +218,7 @@ public void commit() {
                 }
                 // Try to build an index from results
                 MemDocIndex docIndex=null;
-                if(aq.getQuery()!=null&&numSlots>MEM_INDEX_THRESHOLD) {
+                if(aq.getQuery()!=null&&memoryIndexThreshold>=0&&numSlots>memoryIndexThreshold) {
```

Comment:
I remembered why this check is useful - you can disable this feature with memoryIndexThreshold=-1. Without it you'd need to set memoryIndexThreshold to some very large number.